configure.ac

```diff
@@ -224,12 +224,48 @@ then
 fi
 AC_SUBST([ENABLED_ASM])
 
+ENABLED_FIPS="no"
+# wolfEngine Options
+# This requires a FIPS source code bundle.
+AC_ARG_ENABLE([engine],
+    [AS_HELP_STRING([--enable-engine],[Enable wolfEngine options (default: disabled)])],
+    [ ENABLED_WOLFENGINE=$enableval ],
+    [ ENABLED_WOLFENGINE=no ]
+    )
+
+AS_CASE([$ENABLED_WOLFENGINE],
+    [no],[
+        ENABLED_WOLFENGINE="no"
+    ],
+    [disabled],[
```

Comment:
I think you can get rid of this option. It's redundant with "no," and we don't provide "disabled" as a valid option for any other configure options that I'm aware of.

Reply:
The fips options has it. 
```
    [disabled],[
        FIPS_VERSION="disabled"
        ENABLED_FIPS="no"
    ],
```